doc/api/stream.md

```diff
@@ -122,23 +122,22 @@ const server = http.createServer( (req, res) => {
   req.setEncoding('utf8');
 
   // Readable streams emit 'data' events once a listener is added
-  req.on('data', (chunk) => {
+  req.on('data', chunk => {
```

Comment:
nit: our preferred code style in core is to require parens around arguments in arrow functions.